app/routes/docs/schema-browser.tsx

```diff
@@ -5,127 +5,75 @@
  *
  * SPDX-License-Identifier: NASA-1.3
  */
-import { Link, NavLink, Outlet, useLoaderData } from '@remix-run/react'
-import { Icon } from '@trussworks/react-uswds'
-import dirTree from 'directory-tree'
+import type { DataFunctionArgs } from '@remix-run/node'
+import { json, redirect } from '@remix-run/node'
+import { Link, Outlet, useLoaderData, useParams } from '@remix-run/react'
+import { CardBody, CardHeader, Icon } from '@trussworks/react-uswds'
 import { useState } from 'react'
 
-import { SideNav, SideNavSub } from '~/components/SideNav'
-import { feature } from '~/lib/env.server'
+import DetailsDropdownButton from '~/components/DetailsDropdownButton'
+import DetailsDropdownContent from '~/components/DetailsDropdownContent'
+import { publicStaticShortTermCacheControlHeaders } from '~/lib/headers.server'
+import { getVersionRefs } from '~/lib/schema-data'
 
-// Schema treeItem
-type SchemaTreeItem = {
-  name: string
-  path: string
-  children?: SchemaTreeItem[]
-}
+export async function loader({
+  params: { version, '*': path },
```

Comment:
How will this route ever have params? The route module name does not contain any splats.

Reply:
Interesting.